.github/workflows/tests.yml

```diff
@@ -20,18 +25,36 @@ jobs:
         with:
           neovim: true
           version: nightly
+
+      - name: Install C/C++ Compiler
+        uses: rlalik/setup-cpp-compiler@master
+        if: startsWith(matrix.os, 'windows')
+        with:
+          compiler: clang-latest
+
+      - name: Install MSVC Compiler Toolchain
+        if: startsWith(matrix.os, 'windows')
+        uses: ilammy/msvc-dev-cmd@v1
```

Comment:
I'd tend to pin those two to a specific commit - see https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions#using-third-party-actions